packages/metrics/src/api.ts

```diff
@@ -21,16 +21,14 @@ import { logger } from '@opencrvs/commons'
 
 export function fetchFHIR<T = any>(
   suffix: string,
-  authHeader: IAuthHeader,
   method = 'GET',
   body?: string
 ) {
   const url = [fhirUrl.replace(/\/$/, ''), suffix].join('/')
   return fetch(url, {
     method,
     headers: {
-      'Content-Type': 'application/fhir+json',
-      ...authHeader
+      'Content-Type': 'application/fhir+json'
```

Comment:
Ok, here starts some not-very-major but many-in-numbers changes start. I removed the way we send auth tokens to Hearth as it doesn't use it. for anything. More of a risk than a benefit if nothing in the receiving end uses it

Reply:
Only token removals in this file.